test/testLiveAsTemplate.js

```diff
@@ -124,6 +124,10 @@ describe('CLI interface, oghliner as a template', function() {
     .then(function() {
       assert.doesNotThrow(fse.statSync.bind(fse, 'index.html'));
       assert.doesNotThrow(fse.statSync.bind(fse, 'offline-worker.js'));
+    })
+    .catch(function(err) {
+      console.log('Error: ' + err);
+      throw err;
```

Comment:
Is this (and the one below) just for testing, or is it permanent? And if the latter, doesn't it cause duplicate messages to get written to the terminal, one to stdout, the other to stderr?